index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Portfolio Homepage</title>
+</head>
+
+<body>
+  <div id="toolbar">
+    <div id="logoPlaceholder">
+      <p id="logoText">Welcome</p>
+    </div>
+    <input id="headerButton" type="image" src="Images/Enabled.png" alt="Menu">
+  </div>
+  <div id="headline">
+    <div class="primaryText" id="primaryText">
+      <p class="primaryText" id="primaryName">
+        Hey there. I'm Ricardo
+      </p>

```

Comment:
Since this is the heading of the page, kindly make use of the `h1` tag. It is important for every homepage to have a heading represented with the `h1` tag on every website.